src/styles/sidebar/_base.scss

```diff
@@ -34,4 +34,9 @@
   p {
     @apply hyphens-auto;
   }
+
+  html[data-rounded-corners] {
+    --h-border-radius: 0.25rem;
+    --h-border-radius-lg: 0.5rem;
+  }
```

Comment:
I have added this to the sidebar styles only, because the only visible element in the annotator already has a hardcoded `rounded-[4px]` class, which is the new value we want for when the feature flag is enabled.

So basically, we can treat it as always enabled for the annotator. I will replace that `rounded-[4px]` with `rounded` and update the rounded corners for the annotator tailwind config only on a follow-up PR.